build.gradle

```diff
@@ -95,9 +95,19 @@ dependencies {
     compile 'org.apache.commons:commons-lang3:3.5'
     compile 'javax.validation:validation-api:1.1.0.CR2'
     compile 'joda-time:joda-time:2.2'
+    compile 'commons-io:commons-io:2.4'
+    compile 'org.eclipse.jgit:org.eclipse.jgit.archive:4.5.0.201609210915-r'
 }
 
-// This task is used to generate Eiffel events from json schema using jsonschema2pojo plugin.
+/*Gradle task to execute  java class*/
+task runJava(type:JavaExec){
+	main 'com.ericsson.eiffel.remrem.semantics.clone.CloneRemoteRepository'
```

Comment:
It would have been good to have a real gradle task here. Then we could specify the parameters in the build.gradle instead of hiding them in a properties file.